### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (412)
Screenshot (413)
Screenshot (414)
This is my landing page, introduction page and color-scheme page, please review it!!

### CODE REVIEW

File: assets/common-css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+.navbar-container{
+    padding: 20px;
```
Comment (Hamdrive) [inline assets/common-css/navbar.css:3]: Prefer to use `rem` for larger than 10px 
and also better to use CSS variables, like you have for the below code as well

File: assets/common-css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    justify-content: center;
+    align-items: center;
+}
+.lib-name{
+    text-decoration: underline var(--secondary-color);
+    /* font-size: 1.3rem; */
+    color: var(--primary-color);
+    font-size: 1.2rem;
+}
+a{
```
Comment (Hamdrive) [inline assets/common-css/navbar.css]: Prefer to use classnames over elements, for specificity reasons
ALso have a `reset.css` file wherein you set such defaults like `textdecoration` etc

File: assets/common-css/navbar.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    color: var(--secondary-color);
+}
+.list-container{
+    display: flex;
+    margin-left: auto;
+}
+.link{
+    display: inline-block;
+    margin: 0px 10px;
+}
```
Comment (Hamdrive) [inline assets/common-css/navbar.css:34]: Add a new empty line here to remove this warning

File: assets/common-css/sidebar.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    /*  */
+}
+.doc-topic, .components{
+    font-size: 1.2rem;
+    color: var(--font-color);
+    /* text-decoration-line: underline; */
+}
+
+ul{
+    list-style-type: none;
```
Comment (Hamdrive) [inline assets/common-css/sidebar.css:13]: Place in a `reset.css`

File: components/color/color.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    font-size: 1.7rem;
+}
+.color-head{
+    margin-bottom: 1.3rem;
+}
+.color-intro{
+    margin: -5px;
+}
+.color-content{
+    padding: 20px;
```
Comment (Hamdrive) [inline components/color/color.css]: Like mentioned above, prefer to use rems, and define size variables in another css file and use those variables here

File: document-css/doc.css [hunk 6]
```diff
@@ -0,0 +1,43 @@
+@import "../assets/common-css/sidebar.css";
+@import "../assets/common-css/navbar.css";
+@import "../assets/common-css/containers.css";
+
+:root{
```
Comment (Hamdrive) [inline document-css/doc.css]: keep root properties in its own css file

File: landing-page/style.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
+@import "./body.css";
+@import "../assets/common-css/navbar.css";
+@import "../assets/common-css/containers.css";
+@import "../document-css/doc.css";
+
+*{
+    box-sizing: border-box;
+}
+html{
+    font-family: 'Montserrat', sans-serif;
```
Comment (Hamdrive) [inline landing-page/style.css]: Already defined in `body.css` so this is redundant

**Review by Hamdrive — COMMENTED** [review]
Overall code looks good, great work so far
Improve on file structure and apply good practices suggested

**Comment by hinanshisuthar** [comment]
I would suggest you to change your font color of your navigation text. It would be more readable.

File: components/color/color.css [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/sidebar.css";
+
+.color-intro, .color-head{
+    color: var(--font-color);
+    font-size: 1.7rem;
+}
+.color-head{
+    margin-bottom: 1.3rem;
+    margin-top: 30px;
```
Comment (ShreyashMahajan) [inline components/color/color.css]: try to use rem for more than 10px

File: components/text-utilities/text-uti.css [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+    padding: 20px;
+    font-size: 19px;
+    color: var(--primary-color);
+}
+.font-fam-name{
+    color: var(--font-color)
+}
+.different-texts{
+    display: flex;
+    flex-basis: 400px;
```
Comment (ShreyashMahajan) [inline components/text-utilities/text-uti.css]: try to use rem wherever possible

File: components/text-utilities/text-uti.css [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+    padding: 20px;
+    background-color: #dbe0e7;
+    gap: 10px;
+    border-radius: 5px;
+    margin-top: 10px;
+}
+.head-sentence, .small-sentence{
+    color: var(--primary-color);
+}
+.small-sentence{
```
Comment (ShreyashMahajan) [inline components/text-utilities/text-uti.css:41]: for small you can also use "sm".

File: components/text-utilities/text-uti.css [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+}
+.xlarge-gray-text{
+    color: rgb(95, 95, 95);
+    font-size: x-large;
+}
+.large-centered-text, .xlarge-centered-text{
+    color: var(--primary-color);
+    text-align: center;
+}
+.large-centered-text{
```
Comment (ShreyashMahajan) [inline components/text-utilities/text-uti.css:69]: for large "lg" can be used.

**Review by ShreyashMahajan — COMMENTED** [review]
code naming and structure of the website are amazing. only some minor changes are there.

**Pull request merged**